src/main/java/org/isf/opd/gui/OpdBrowser.java

```diff
@@ -845,30 +893,35 @@ public int getColumnCount() {
 		public Object getValueAt(int r, int c) {
 			Opd opd = pSur.get(pSur.size() - r - 1);
 			Patient pat = opd.getPatient();
+			int i = 0;
 			if (c == -1) {
 				return opd;
-			} else if (c == 0) {
+			} else if (c == i) {
 				return opd.getCode();
-			} else if (c == 1) {
+			}  else if (c == ++i) {
```

Comment:
I personally like having the value there instead of `++1` so I don't have to compute where I am in the list.

Reply:
there's no need to compute, as far you see the actual column content... instead it is easier to add columns at any place without renumbering all :)

Reply:
for this we will agree a solution to apply all over JTables, thanks!